src/nodes.cpp

```diff
@@ -65,31 +76,55 @@ Selection NodePopulation::matchAttributeValues(const std::string& name, const T
     }
 }
 
+template <typename T>
+Selection NodePopulation::matchAttributeValues(const std::string& name, const T value) const {
+    std::vector<T> values{value};
+    return matchAttributeValues<T>(name, values);
+}
+
 template <>
-Selection NodePopulation::matchAttributeValues<std::string>(const std::string& name,
-                                                            const std::string value) const {
-    // enum attribute, need to look up in the @library
+Selection NodePopulation::matchAttributeValues<std::string>(
+    const std::string& name, const std::vector<std::string>& values) const {
     if (enumerationNames().count(name) > 0) {
```

Comment:
For enumeration values, this would be a *really convenient* location to do some glob resolving :)